tests/acceptance/features/coreApiWebdavLocksUnlock/unlock.feature

```diff
@@ -1,5 +1,8 @@
 @api @issue-1284
-Feature: UNLOCK locked items
+Feature: Unlock locked items
+  As a user
+  I want to unlock the resources set by myself
+  So that other users can make changes on the resources
```

Comment:
```suggestion
  As a user
  I want to unlock the resources previously locked by myself
  So that other users can make changes to the resources
```